tests/test/scheduler/test_mpi_world.cpp

```diff
@@ -137,7 +137,9 @@ TEST_CASE_METHOD(MpiBaseTestFixture, "Test cartesian communicator", "[mpi]")
     for (int i = 0; i < dims[0]; i++) {
         for (int j = 0; j < dims[1]; j++) {
             std::vector<int> coords = { i, j, 0 };
-            int rank, src, dst;
+            int rank;
```

Comment:
Changes in this file are only fixes of clang tidy warnings (now that I see them). I only include them here as it is related to the PR. Happy to discard them though.